modules/monitoring-creating-cluster-monitoring-configmap.adoc

```diff
@@ -6,7 +6,7 @@
 [id="creating-cluster-monitoring-configmap_{context}"]
 = Creating a cluster monitoring config map
 
-To configure core {product-title} monitoring components, you must create the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+You can configure the Cluster Monitoring Operator (CMO) by creating the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures core {product-title} monitoring components.
```

Comment:
```suggestion
You can configure the core {product-title} monitoring components by creating the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures core {product-title} monitoring components.
```
we configured core monitoring components, example: alertmanager/prometheus, not for CMO